### PR TITLE
fix(collector-schedule-form): refactor schedule form

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

**Review by skdud4659 — APPROVED** [review]
GOOD! thx!

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorScheduleSection.vue [hunk 1]
```diff
@@ -1,17 +1,15 @@
 <template>
     <p-pane-layout>
         <section-header :title="$t('INVENTORY.COLLECTOR.DETAIL.SCHEDULE')"
-                        :edit-mode="state.isEditMode"
+                        :edit-mode="state.isEditMode || !collectorFormState.schedulePower"
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorScheduleSection.vue:4]: suggestion: I think this logic is pretty duplicated with the watcher in the bottom of this file.

**Review by WANZARGEN — COMMENTED** [review]
I think there are duplicated logics so I left one comment! :) Thanks for your quick update!

**Review by WANZARGEN — APPROVED** [review]
LGTM!!!!!

**Pull request merged**